crates/sozo/ops/src/migration/mod.rs

```diff
@@ -145,11 +149,8 @@ async fn update_manifests_and_abis(
     let ui = ws.config().ui();
     ui.print("\n✨ Updating manifests...");
 
-    let deployed_path = manifest_dir
-        .join(MANIFESTS_DIR)
-        .join(DEPLOYMENTS_DIR)
-        .join(chain_id)
-        .with_extension("toml");
+    let deployed_path = build_deployed_path(manifest_dir, chain_id, "toml");
+    let deployed_path_json = build_deployed_path(manifest_dir, chain_id, "json");
```

Comment:
do we plan to always write the manifest in `json` format along with `toml`?

Reply:
For now yes, as the client always need this file. So in the pipeline from FE devs, I would say for the moment it is worth having it by default as users are used to it.
Sounds good?